mmv1/products/dataplex/Datascan.yaml

```diff
@@ -794,3 +899,26 @@ properties:
               - !ruby/object:Api::Type::String
                 name: 'end'
                 description: Value that marks the end of the range.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'postScanActionsResult'
+        decription: |
+           The result of post scan actions.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'bigQueryExportResult'
+            description: |
+              The result of BigQuery export post scan action.
+            properties:
+              - !ruby/object:Api::Type::Enum
+                name: 'state'
+                decription: |
+                  Execution state for the BigQuery exporting.
+                values:
+                  - STATE_UNSPECIFIED
+                  - SUCCEEDED
+                  - FAILED
+                  - SKIPPED
+              - !ruby/object:Api::Type::String
+                name: 'message'
+                description: |
+                  Additional information about the BigQuery exporting.
```

Comment:
This whole block seems like it should be an [output](https://github.com/GoogleCloudPlatform/magic-modules/blob/50087bde9e5ae445fac5fa643c8f81c195ded68a/mmv1/api/type.rb#L39) field.

Reply:
Note that we are looking to remove the DataProfileResult and DataQualityResult in this PR: https://github.com/GoogleCloudPlatform/magic-modules/pull/8583#issuecomment-1670332184

Based on how that discussion goes, we should look at only adding the export and rule name fields in the spec, and not in the result.

@evasharma

Reply:
Thanks removed the fields added in this PR, the rest of the Result field to be removed in @saurabh-net PR.